sound/snapcast/Makefile

```diff
@@ -0,0 +1,127 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=snapcast
+BASE_VERSION:=0.28.0
+PKG_SOURCE_DATE:=2024-04-15
+PKG_SOURCE_VERSION:=5fa01b8a49d842ed368138ee071cdf1c34be7a64
```

Comment:
why not just use the version here? something like

```
PKG_VERSION:=0.28.0
PKG_SOURCE_VERSION:=v$(PKG_VERSION)
```

as is done elsewhere.

Reply:
> why not just use the version here?

[Snapcast has](https://github.com/badaix/snapcast/branches) `master` branch for release versions and `develop` branch for nightly builds.

In the older days there was too much commit between the two and the development version was stable enough to use it.

That's why I wanted to include the exact commit and commit date from what the actual version was built.

Sure, **badaix** releases Snapcast often these days, so your suggestion could work too.
I would stick with this advanced versioning, but if you ask me to change it to the simple versioning, I would do it.

Reply:
I'd like that. If needed, SOURCE_DATE can be revisited.